### PR TITLE
Build/#35 Vuepress for documentation

### PR DESCRIPTION
**Pull request by leomp12**
# Related issue
https://github.com/DivanteLtd/storefront-ui/issues/35

# Scope of work
- Added Vuepress to npm dev dependencies;
- General configuration for building documentation from `/docs` folder with Vuepress;
- Minor fixes on `docs/customization.md` file to prevent error when building documentation (HTML must be escaped);
- Live sample deployed from my fork https://festive-pike-0cf80f.netlify.com/;

## Must change
1. Replace the Vue logo with Storefront UI logo at `docs/.vuepress/public/logo.png` (I haven't this logo image), you may also want to change the `hero.png` image (at the same dir), shown on homepage;
2. Change the features listed on homepage by editing the file `docs/README.md` (as I think I'm not the best person for it, I've just kept the examples from Vuepress docs site);

## Deploying
I suggest deployment with Netlify, just following the same settings from [Vuepress docs](https://vuepress.vuejs.org/guide/deploy.html#netlify), also needs to enable **Pretty URL** feature, from Netlify dashboard at _Settings > Build & deploy > Post processing > Asset optimization_.

tl;dr: taking advantage of the fact that Netlify would already be installed in the repository, deployment with it would also solve [this issue](https://github.com/DivanteLtd/storefront-ui/issues/34) for free and with no work required (just pointing the domain, of course...) :smile:

### CODE REVIEW

**Comment by filrak** [comment]
@leomp12 would you be able to update this PR? Now it's time to merge it in ;)

**Comment by leomp12** [comment]
Yup

**Comment by leomp12** [comment]
@filrak I think it's ready for a first version:
https://festive-pike-0cf80f.netlify.com/

At `docs/introduction.md` I've linked _docs.storefrontui.io_, you may change it if planning to use another subdomain.
I've also edited introduction following text on storybook home.

**Comment by filrak** [comment]
Looks awesome! thanks
